docs/02-for-app-authors/07-updates.md

```diff
@@ -0,0 +1,17 @@
+# Updates
+
+Flathub builds and publishes app updates as soon as possible after a change is made to an app's manifest. Depending on the build backlog, this can be anywhere from minutes to hours.
```

Comment:
That's not 100% true, we still have the three hour publishing delay

Reply:
Ah, thanks! I suppose this is a good reason to document this stuff. 😄

Reply:
It is mentioned in https://docs.flathub.org/docs/for-app-authors/maintanance#buildbot but doesn't hurt to mention it here too